main/pom.xml

```diff
@@ -43,7 +46,7 @@
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>build-helper-maven-plugin</artifactId>
-	<version>${build-helper-maven-plugin.version}</version>
+	    <version>${build-helper-maven-plugin.version}</version>
```

Comment:
It looks like you have some spurious indentation changes in this file. It would be good to avoid them. You can see the full list here: https://github.com/OpenRefine/OpenRefine/pull/4978/files

Reply:
Oh! I did not notice that. I will get to working on this